pt-br/tutorials/tour/automatic-closures.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: tutorial
+title: Automatic Type-Dependent Closure Construction
+
+disqus: true
+
+tutorial: scala-tour
+num: 30
+tutorial-next: annotations
+tutorial-previous: operators
+language: pt-br
+---
+
+O Scala permite nomes de funções sem parâmetros como parâmetros de métodos. Quando um tal método é chamado, os parâmetros reais para nomes de função sem parâmetros não são avaliados e uma função nula é passada em vez disso, tal função encapsula a computação do parâmetro correspondente (isso é conhecido por avaliação *call-by-name*).
```

Comment:
In the original it says: `Scala allows parameterless function names as parameters of methods`. I don't understand why the phrase needs the "name" part. WDYT? Could we drop "nomes" in `Scala permite nomes de funções sem parâmetros como parâmetros de métodos`?